README.md

```diff
@@ -97,6 +97,7 @@ params.adjustments.fajr = 2
 | --------- | ----------- |
 | method    | Member of CalculationMethod enum |
 | fajrAngle | Angle of the sun used to calculate Fajr |
+| maghribAngle | Angle of the sun used to calculate Maghrib |
```

Comment:
Let's clarify with: `Angle of the sun used to calculate Maghrib for some Calculation Methods`